backend/tests/mw_drw_proxy/proxy.py

```diff
@@ -0,0 +1,165 @@
+# Copyright (c) The Diem Core Contributors
+# SPDX-License-Identifier: Apache-2.0
+
+import json
+import os
+import typing
+
+from flask import Flask, request
+import logging
+
+from diem import identifier
+
+from tests.e2e_tests import UserClient, create_test_user
+from tests.mw_drw_proxy.helpers import (
+    DEFAULT_KYC_INFO,
+    convert_offchain_to_user_info,
+    convert_user_info_to_offchain,
+    error_response,
+)
+
+APP_NAME = __name__.split(".")[0]
+LOG = logging.getLogger(APP_NAME)
+logging.basicConfig(level=logging.INFO)
+
+app = Flask(APP_NAME)
+
+STATE: typing.Dict[str, UserClient] = {}
+
+DRW_URL_PREFIX = os.getenv("DRW_URL_PREFIX", "https://demo-wallet.diem.com")
```

Comment:
we should use same env variable e2e test is using, and proxy to demo-wallet.diem.com is not a good idea as it can't connect to mini-wallet stub offchain API, has to be local.